modules/fedimint-dummy-common/src/lib.rs

```diff
@@ -94,12 +91,11 @@ impl ModuleCommon for DummyModuleTypes {
     type ConsensusItem = DummyConsensusItem;
 }
 
-plugin_types_trait_impl!(
```

Comment:
We need to delete `plugin_types_trait_impl` BTW. Only `plugin_types_trait_impl_common` should be needed now.